src/modules/order/containers/PaymentDetail/hooks/useFormNormalization.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from 'react';
+
+const useFormNormalization = () => {
+  const normalizeCardNumber = useCallback((value: string) => {
```

Comment:
I'm not sure if a hook is really needed here, helper (util) func might be sufficient, but ok)